Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2743,10 +2743,43 @@ bool CGContextIsPointInPath(CGContextRef context, bool eoFill, CGFloat x, CGFloa
 struct __CGBitmapContext : CoreFoundation::CppBase<__CGBitmapContext, __CGContext> {
     woc::unique_cf<CGImageRef> _image;
 
+    void* _data;
```

Comment:
@aballway what is the value in distinguishing Objective-C and C++ code? Members and instance variables are, for all intents and purposes, identical. A fully-qualified out-of-body member function refers to members just like an Objective-C method refers to ivars.

They have similar lifetime requirements, as well.

our CF stuff is closer to Objective-C due to the reference counting; it's only C++ as a measure of convenience.

Also, our coding convention states

> Use the "_" prefix on all data members of Objective C and C++ classes. This is consistent with Objective C instance variables that are synthesized automatically by the compiler.